### PR TITLE
2023-04-16 release notes

### PR DESCRIPTION
**Pull request by shashankbarsin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-04-16
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+ 
+* Starting on March 21, 2023, traffic to k8s.gcr.io will be redirected to registry.k8s.io, following the [community announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/).
+* Docker container runtime will be retired for Windows nodepools on May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
```
Comment (kaarthis) [inline CHANGELOG.md:10]: I think this Support policy https://techcommunity.microsoft.com/t5/apps-on-azure-blog/azure-kubernetes-upgrades-and-long-term-support/ba-p/3782789  announcement : will catch a lot of eyeballs good to throw it in For non-LTS clusters, each Kubernetes version is supported for 12 months. After 12 months, the minor version will shift to platform support only. Our new platform support policy provides customers with Azure infrastructure support while the cluster is in an n-3 version (where n is the latest supported AKS GA minor version).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+
+### Release notes
+
+* Features
```
Comment (kaarthis) [inline CHANGELOG.md:14]: What about this one ? https://techcommunity.microsoft.com/t5/apps-on-azure-blog/general-availability-for-azure-active-directory-ad-workload/ba-p/3798292

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-04-16
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
```
Comment (kaarthis) [inline CHANGELOG.md:7]: Install OpenCost on AKS https://techcommunity.microsoft.com/t5/apps-on-azure-blog/leverage-opencost-on-azure-kubernetes-service-to-understand-and/ba-p/3796813 
OpenCost has released a special KubeCon EU 2023 build incorporating the above features along with additional community member contributions. To try the build, refer to [Azure OpenCost Integration](https://www.opencost.io/docs/azure-opencost). These features will be generally available in production in OpenCost v103, scheduled for May 2, 2023.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+* Starting on March 21, 2023, traffic to k8s.gcr.io will be redirected to registry.k8s.io, following the [community announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/).
+* Docker container runtime will be retired for Windows nodepools on May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
+
+### Release notes
+
+* Features
+  * [Dual-stack networking (IPv4/IPv6) on kubenet](https://learn.microsoft.com/azure/aks/configure-kubenet-dual-stack?tabs=azure-cli%2Ckubectl) is now generally available.
+
+* Preview Features
+  * [Istio-based service mesh add-on](https://learn.microsoft.com/azure/aks/istio-about) for Azure Kubernetes Service is now available in preview.
```
Comment (kaarthis) [inline CHANGELOG.md:18]: Maybe we can keep this for one more week as its just Kubecon week .  [Stop cluster minor version upgrades on API breaking changes](https://learn.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli) is now available in preview. AKS will fail fast on minor version manual upgrades if it detects usages of deprecated APIs in the target version of the upgrade. This is available when target cluster for upgrade is >= 1.26.0, when the API request for cluster is using API version >= 2023-01-02-preview, and when usage of API breaking changes has been detected in the 12 hours prior to the upgrade.

**Review by qpetraroia — APPROVED** [review]

**Review by kaarthis — COMMENTED** [review]
see my comments

File: CHANGELOG.md (hunk 2 above)
Comment (shashankbarsin) [reply CHANGELOG.md:14]: already covered in previous week's notes

**Pull request merged**